Hardware/Hardware/CardReader/UnderlyingError.swift

```diff
@@ -324,6 +327,12 @@ extension UnderlyingError: LocalizedError {
             return NSLocalizedString("Unable to connect to reader - the reader has a critically low battery - charge the reader and try again.",
                                      comment: "Error message the card reader battery level is too low to connect to the phone or tablet.")
 
+        case .bluetoothDenied:
+            return NSLocalizedString(
+                "hardware.cardReader.underlyingError.bluetoothDenied",
+                value: "This app needs permission to access Bluetooth to connect to a card reader, please change the privacy settings if you wish to allow this.",
```

Comment:
```suggestion
                value: "This app needs permission to access Bluetooth to connect to your card reader. You can grant permission in the system's Settings app, in the Woo section.",
```

A button which took them to the right place would obviously be better, but if you use wording like this it may help steer some people to the right place.

Reply:
Thanks for the suggestion! 

Yes, I agree that having a button to go straight into the privacy settings would be more convenient. I'll check if it's a quick improvement and can make another PR, or just create another improvement task in the backlog.